src/wallet.h

```diff
@@ -111,13 +111,33 @@ class CWallet : public CCryptoKeyStore, public CWalletInterface
     CWallet()
     {
         SetNull();
+        
+        fStakeForCharity = true;
+    	nStakeForCharityPercent = 1;
+    	CBitcoinAddress StakeForCharityAddress = "BRfGmqCg6kKBwWTzzMVYoq3BXp2t6oWAzx";
```

Comment:
nope, this can be disabled by default, or enabled and sent to an official dev account, exchanges shouldn't argue about pos rewards at all, plus it can be disabled via rpc